src/services/tourettes.js

```diff
@@ -62,12 +62,22 @@ function getProfanities(amount) {
   return profanityReply.join(' ');
 }
 
+function isInt(data) {
```

Comment:
Well, it specifically checks if the number is an integer, it would be false if it were a float e.g. I think more of the English word rather than types.
So it would be less descriptive in my mind to use number.
But anyways, isInteger says more than isInt.